<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateEditor</class>
 <widget class="QDialog" name="TemplateEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор шаблона</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Имя шаблона:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="templateNameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Название сериала:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="seriesTitleEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>URL RSS-фида:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="rssUrlEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Студия анимации:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="animationStudioEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Автор субтитров:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="subAuthorEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Язык оригинала (код):</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="originalLanguageEdit">
         <property name="text">
          <string>jpn</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Название главы эндинга:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="endingChapterNameEdit">
         <property name="placeholderText">
          <string>например, Ending</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Время начала эндинга:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="endingStartTimeEdit">
           <property name="placeholderText">
            <string>например, 0:22:22.26</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="useManualTimeCheckBox">
           <property name="text">
            <string>Вручную</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Режиссер дубляжа:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="directorEdit"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Звукорежиссер:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="soundEngineerEdit"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Разметка:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="timingAuthorEdit"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_signs_author">
         <property name="text">
        <string>Локализация надписей:</string>
        </property>
        </widget>
       </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="signsAuthorEdit"/>
      </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Сборка релиза:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="releaseBuilderEdit"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_17_2">
         <property name="text">
          <string>Формат аудио для сборки:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QComboBox" name="targetAudioFormatComboBox">
         <item>
          <property name="text">
           <string>aac</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>flac</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_type">
         <property name="text">
          <string>Тип озвучки:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QComboBox" name="voiceoverTypeComboBox">
         <item>
          <property name="text">
           <string>Дубляж</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Закадр</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QCheckBox" name="createSrtMasterCheckBox">
         <property name="toolTip">
          <string>После основной сборки создать дополнительный MKV с несжатым звуком и SRT-субтитрами (без надписей)</string>
         </property>
         <property name="text">
          <string>Создать дополнительную сборку с SRT (мастер-копия)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCast">
      <attribute name="title">
       <string>Каст и стили</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Список актеров (каждый с новой строки):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="castEdit"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Стили/актеры для надписей (каждый с новой строки):</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPlainTextEdit" name="signStylesEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>3</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectStylesButton">
           <property name="text">
            <string>Выбрать из файла субтитров...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="sourceHasSubtitlesCheckBox">
         <property name="text">
          <string>В исходном MKV есть дорожка русских субтитров</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="generateTbCheckBox">
         <property name="text">
          <string>Генерировать Титры Благодарности (ТБ)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTbStyles">
      <attribute name="title">
       <string>Стили ТБ</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tbStylesTable">
         <column>
          <property name="text">
           <string>Имя стиля</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PlayResX</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Теги</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Margin L</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Margin R</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Margin V</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="addTbStyleButton">
           <property name="text">
            <string>Добавить стиль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeTbStyleButton">
           <property name="text">
            <string>Удалить выбранный</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Стиль по умолчанию:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="defaultTbStyleComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPosts">
      <attribute name="title">
       <string>Шаблоны постов</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="helpButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Открыть справку по плейсхолдерам и форматированию</string>
           </property>
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Telegram (MP4 пост):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="postTgMp4Edit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Telegram (MKV пост):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="postTgMkvEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>VK (Основной пост):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="postVkEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>VK (Комментарий):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="postVkCommentEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPublication">
      <attribute name="title">
       <string>Публикация</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Название для постов:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="seriesTitleForPostEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Всего серий в сезоне:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="totalEpisodesSpinBox">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Путь к постеру:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="posterPathEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browsePosterButton">
           <property name="text">
            <string>Обзор...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Ссылки для загрузки видео (каждая с новой строки):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="uploadUrlsEdit">
         <property name="placeholderText">
          <string>https://anilib.me/upload...
https://anime-365.ru/upload...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TemplateEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TemplateEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
